docs/personal.md

```diff
@@ -5,16 +5,27 @@ title: Personal
 
 ## College
 
-I went to the [University of Nebraska at Omaha](https://www.unomaha.edu/) and majored in music performance but eventually dropped out due to conflict with work at the time.
+I attended the [University of Nebraska at Omaha](https://www.unomaha.edu/) and majored in woodwind performance but eventually dropped out due to conflict with work at the time.
 
-In general, I do not think that people need to have a college degree in order to have good opinions or to know what they are talking about. I'd say that over the past several years of debate and conversation my respect for college-educated individuals has significantly dropped. That being said, a formal education certainly doesn't hurt. I think laymen should generally defer to the consensus of experts. A good example of this can be found in [my discussion with Vegan Gains](https://www.youtube.com/watch?v=9yK-lO98scI).
+In general, I do not think that people need to have a college degree in order to have good opinions or to know what they are talking about. Over the past several years of debate and conversation my respect for college-educated individuals has significantly dropped. That being said, a formal education certainly doesn't hurt. I think laymen should generally defer to the consensus of experts. A good example of this can be found in [my discussion with Vegan Gains](https://www.youtube.com/watch?v=9yK-lO98scI).
 
 ## My Family
 
-I have a son called Nathaniel with my ex-girlfriend, Rachel. My ex-girlfriend and I broke up a long time ago because we had an incredibly toxic relationship, but we are on very good terms today and get along well with each other. I have an ex-wife as well, but she is not my ex-girlfriend and she is not the mother of my child.
+I have a son called Nathaniel with my ex-girlfriend, Rachel. My ex-girlfriend and I broke up a long time ago due to our toxic relationship, but we are on good terms today and get along well with each other. I have an ex-wife as well, but she is not the mother of my child.
 
 ## My Relationship
 
 [Melina](https://www.twitch.tv/melina) is my wife. We met in New Zealand when she was 20 years old and I was 30 years old.
 
-Melina and I are currently in an open/poly relationship. We treat each other as primary partners, though we may pursue other sexual/romantic relationships as well.
+Melina and I are currently in an open/poly relationship. We treat each other as primary partners, though we may pursue other sexual/romantic relationships as well.8
```

Comment:
good one bonnell "well.8"

Reply:
poor quality merge request please learn to code and try again